elasticsearch.html.md.erb

```diff
@@ -11,51 +11,32 @@ Elasticsearch is a powerful open-source search engine for full text search.
 
 ## <a id='integrating-your-service'></a> Integrating the Service With Your App
 
-After the [creation](../devguide/services/managing-services.html#create) of the service and the [binding](../devguide/services/application-binding.html#bind) of the service to the application, the environment variable [VCAP_SERVICES](../devguide/deploy-apps/environment-variable.html#VCAP-SERVICES) is created. This variable stores information about the credentials as shown here:
+After the [creation](../devguide/services/managing-services.html#create) of the service and the [service key](../devguide/services/service-keys.html#create), you'll find all required credentials in this service key, as shown here:
 
 ```json
 {
- "VCAP_SERVICES": {
-  "elasticsearch": [
-   {
-    "binding_name": null,
-    "credentials": {
-     "full_access_password": "password",
-     "full_access_username": "username",
-     "host": "https://d15c5caa5a3147ed9b444e4694162b3f.elasticsearch.lyra-836.appcloud.swisscom.com",
-     "kibana_read_only_password": "password",
-     "kibana_read_only_username": "username",
-     "ece_kibana_dashboard_only_username": "username",
-     "ece_kibana_dashboard_only_password": "password",
-     "kibana_system_password": "password",
-     "kibana_system_username": "username",
-     "logstash_system_password": "password",
-     "logstash_system_username": "username"
-    },
-    "instance_name": "elasticsearch-service",
-    "label": "elasticsearch",
-    "name": "elasticsearch-service",
-    "plan": "xxsmall",
-    "provider": null,
-    "syslog_drain_url": null,
-    "tags": [
-     "elasticsearch"
-    ],
-    "volume_mounts": []
-   }
-  ]
- }
+  "host": "https://d15c5caa5a3147ed9b444e4694162b3f.elasticsearch.lyra-836.appcloud.swisscom.com",
+  "kibana_system_username": "username",
+  "kibana_system_password": "password",
+  "logstash_system_username": "username",
+  "logstash_system_password": "password",
+  "kibana_read_only_username": "username",
+  "kibana_read_only_password": "password",
+  "kibana_dashboard_only_username": "username",
+  "kibana_dashboard_only_password": "password",
+  "full_access_username": "username",
+  "full_access_password": "password"
 }
 
 ```
 
 ## <a id='privileges'></a> User Privileges
 
-Each [binding](../devguide/services/application-binding.html#bind) comes with a set of four users with different privileges.
+Each [service key](../devguide/services/service-keys.html) or [binding](../devguide/services/application-binding.html#bind) to the elasticsearch-service comes with a set of five users with different privileges.
```

Comment:
I would get rid of the numeric mention of the amount of credentials. Chances are high that this is wrong very often.